packages/engine/Source/Shaders/Builtin/Functions/sampleOctahedralProjection.glsl

```diff
@@ -68,7 +68,7 @@ vec3 czm_sampleOctahedralProjectionWithFiltering(sampler2D projectedMap, vec2 te
  * @returns {vec3} The color of the cube map at the direction.
  */
 vec3 czm_sampleOctahedralProjection(sampler2D projectedMap, vec2 textureSize, vec3 direction, float lod, float maxLod) {
-    float currentLod = floor(lod + 0.5);
+    float currentLod = floor(lod);
```

Comment:
I agree that the fix is resulting in better appearances for sure. But any idea why there was a `0.5` increment in the first place?

Reply:
I think this dates back to before we were interpolating the mip level. In [this previous version](https://github.com/CesiumGS/cesium/blame/1a781a85903a83ebc598c815341b4602de707c70/Source/Shaders/Builtin/Functions/sampleOctahedralProjection.glsl) we were simply rounding to the nearest mip, which required the `0.5` increment. https://github.com/CesiumGS/cesium/commit/006d8597853638ac2bdb4148ac906b9a3f50df15 added trilinear filtering but neglected to remove the `0.5` when selecting the "currentLod".